### DISCUSSION TITLE
Not working in Android Maui Blazor

### DISCUSSION THREAD

**Discussion by jcmontoya** [sabitertan/BlazorBarcodeScanner#36] [Q&A]
I am testing with a Maui Blazor App. The scanner works on windows but not in Android. I granted access to the camera in the phone. Any ideas what I am missing? thanks in advance.

**Comment by cipeu** [comment]
I got the same problem
I already set the CAMERA permission in the android manifest.

**Reply by sabitertan** [reply]
This library targets web applications, please use MAUI specific library like https://github.com/Redth/ZXing.Net.Maui .
